tests/unit/test.utils.js

```diff
@@ -689,3 +691,44 @@ describe('isCompatible', () => {
     expect(isCompatible(bcd, 'runtime.getURL', 44, 'firefox')).toBe(false);
   });
 });
+
+describe('errorParamsToUnsupportedVersionRange', () => {
+  const testCases = [
+    [{ min_manifest_version: 3 }, '< 3'],
+    [{ max_manifest_version: 2 }, '> 2'],
+    [
+      // This would be actually used anywhere because each
+      // schema validation would be only including one of them
+      // and generate separate validation errors.
+      { min_manifest_version: 2, max_manifest_version: 4 },
+      '< 2 or > 4',
+    ],
+    [{}, ''],
+    [null, ''],
+    [undefined, ''],
+  ];
+  for (const [params, expected] of testCases) {
```

Comment:
Each time we are not using `it.each()` for creating new test cases, we should use `expect.assertions()` I think.

Reply:
:heart: I rewrote this into an `it.each` form